packages/core/test-helpers/src/helpers.ts

```diff
@@ -3,7 +3,7 @@ import { AdapterRequest, Execute } from '@chainlink/types'
 
 export function assertError(statusCode: any, data: any, expectedJobId: any) {
   expect(statusCode.actual).toEqual(statusCode.expected)
-  expect(data.jobRunID).toEqual(expectedJobId)
+  expect(data.jobRunID).toEqual(expectedJobId || '1')
```

Comment:
Want to double check this.  My tests were failing because the validator object exported from core set the jobId to 1 if it's null.  This test was failing for the Crypto API V1 adapter as well.

Reply:
I think we want to include the jobRunID on the failing tests, so this change shouldn't be necessary imo